sql/postgresql/README.md

```diff
@@ -25,7 +25,73 @@ createdb my_pap_db
 psql --dbname=my_pap_db --file="<pingauthorize-contrib-root>/sql/postgresql/init-policy-database.sql" >/dev/null
 ```
 You should now have an initialized policy database. For instructions on configuring the Policy Editor to connect to
-your PostgreSQL instance, see "[Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)".
+your PostgreSQL instance, see [Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive).
+
+
+### Upgrading an existing policy database to a newer schema
+1. Verify the current version of your policy database using one of the following methods.
+
+**Optional** - Access `my_pap_db` and query the `TAG` column of `my_pap_db.databasechangelog`
+
+```
+psql -U postgres -d my_pap_db
```

Comment:
Another small nit here, but the prerequisites list that the admin should already be signed in as an authenticated PostgreSQL superuser.